From 6183d512e7539033ccfd177d5f5819302d1fda99 Mon Sep 17 00:00:00 2001
From: Lijun Pan <Lijun.Pan@freescale.com>
Date: Wed, 23 Sep 2015 17:06:01 -0500
Subject: [PATCH 234/238] fsl-ifc: fix compilation error when COMPAT not
 enabled

When CONFIG_COMPAT is not enabled for cases when 64K pages
are enabled, there are a series of include dependencies that
result in some definitions in sched.h that get missed (e.g.
 TASK_NORMAL).  Explictly include sched.h to resolve this.
(This seems to be what other drivers do as well)

Signed-off-by: Lijun Pan <Lijun.Pan@freescale.com>
[Stuart: updated subject and commit message]
Signed-off-by: Stuart Yoder <stuart.yoder@freescale.com>
---
 drivers/memory/fsl_ifc.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/memory/fsl_ifc.c
+++ b/drivers/memory/fsl_ifc.c
@@ -35,6 +35,7 @@
 #include <linux/irqdomain.h>
 #include <linux/of_address.h>
 #include <linux/of_irq.h>
+#include <linux/sched.h>
 
 struct fsl_ifc_ctrl *fsl_ifc_ctrl_dev;
 EXPORT_SYMBOL(fsl_ifc_ctrl_dev);
